android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectsListAdapter.java

```diff
@@ -287,6 +294,10 @@ else if(trans.getNextRequestTime() < nextRetryS || nextRetryS == 0) {
                                                                        NumberFormat.getIntegerInstance().format(hostCredit) :
                                                                        this.activity.getString(R.string.projects_credits_host_and_user, hostCredit, userCredit));
 
+            TextView tvDiskUsage = vi.findViewById(R.id.project_disk_usage);
```

Comment:
Please fix indents

Reply:
This is the correct indent.
Please open full file view (or open more up and down) to see that it correct indent.